css/styles.css

```diff
@@ -0,0 +1,201 @@
+* {
```

Comment:
@AmyMirisola change this to `body {..style..}`, so that they apply to the body tag and it's children only 🔨